spaceship/lib/spaceship/tunes/app_version.rb

```diff
@@ -651,6 +682,31 @@ def setup_screenshots_for(row)
         return result
       end
 
+      # generates the nested data structure to represent screenshots
+      def setup_messages_screenshots_for(row)
+        return [] if row.nil? || row["displayFamilies"].nil?
+
+        display_families = row.fetch("displayFamilies", {}).fetch("value", nil)
+        return [] unless display_families
+
+        result = []
+
+        display_families.each do |display_family|
+          display_family_screenshots = display_family.fetch("messagesScreenshots", {})
+          next unless display_family_screenshots
```

Comment:
I forget why I exactly did that, but I'm pretty sure under certain circumstances `display_family_screenshots` is nil. I fixed the bug specifically in a commit: https://github.com/fastlane/fastlane/pull/6823/commits/355403ec5bff4f9029dc5416650a6eb356e38906